### DISCUSSION TITLE
How to get scroll delta / y offset of content in BottomSheetScrollView?

### DISCUSSION THREAD

**Discussion by VityaSchel** [gorhom/react-native-bottom-sheet#623] [Q&A]
I want to track scrolling of `<BottomSheetScrollView>` and change its content accordingly.  I tried props `onScroll`, `onMomentumScrollEnd` as callbacks, `contentOffset` as shared value, but none of them seems to work. Is there a way to accomplish my goal?

**Comment by ozasadnyy** [comment]
@gorhom @vonovak any suggestions?

**Comment by Karthik-B-06** [comment]
Hey @VityaSchel, were you able to make this work? Even I am trying to track scroll, to trigger scroll based animations. Is there a way to do it? 

@gorhom

**Reply by VityaSchel** [reply]
i have no idea it's been 3 years lol

**Reply by Karthik-B-06** [reply]
Haha, now I am using the `animatedPosition` to work the appear animations for `View` items on each snap points.
